### ISSUE TITLE
A bash script to create everything for the user

### ISSUE THREAD

**Issue by A-Harby** [threefoldtech/quantum-storage#39]
Having the user install Prerequisites packages, install our components (0-db and others), deploy multiple ZDBs, get their information written in Metadata Zdbs Configuration, and also create zstor-default.Toml is a really long and hard process (with a lot of errors).

It would be a lot better for the user to have a bash script to install all the packages and components and create configuration files.

**Comment by Mik-TF** [comment]
Good idea! Once this is done we could use the script to build a Docker image of QSFS  then turn it into flist to deploy on micro VM. With the flist we can make a dashboard app for QSFS!

**Comment by scottyeager** [comment]
Work is ongoing to create a fully automated deployment using Pulumi:

https://github.com/threefoldtech/quantum-storage/tree/pulumi

**Comment by scottyeager** [comment]
An even better way is available with quantumd.

https://github.com/threefoldtech/quantum-storage/tree/master/quantumd
